src/test/java/htsjdk/samtools/reference/AbstractIndexedFastaSequenceFileTest.java

```diff
@@ -27,10 +27,7 @@
 import htsjdk.HtsjdkTest;
 import htsjdk.samtools.SAMException;
 import htsjdk.samtools.seekablestream.SeekableFileStream;
-import htsjdk.samtools.util.CloserUtil;
-import htsjdk.samtools.util.GZIIndex;
-import htsjdk.samtools.util.RuntimeIOException;
-import htsjdk.samtools.util.StringUtil;
+import htsjdk.samtools.util.*;
```

Comment:
I think we should avoid using `*` imports as it makes it harder to see which package an import came from outside of an IDE. There's an IDE setting that will automatically fix this when adding/removing imports.